src/doc/man/cargo-info.md

```diff
@@ -60,7 +60,7 @@ selected based on the Minimum Supported Rust Version (MSRV).
         cargo info serde@1.0.0
 3. Inspect the `serde` package form the local registry:
 
-        cargo info serde --registry my-registry 
```

Comment:
I'm not sure if it’s appropriate to include this change in the same commit. I’m fine with removing it or moving it to a separate commit if needed.